renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+  "enabled": true,
+  "extends": [":semanticPrefixFixDepsChoreOthers"],
+  "schedule": ["before 2am"],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "groupName": "all dependencies",
+      "groupSlug": "all"
+    }
+  ],
+  "rangeStrategy": "auto",
+  "lockFileMaintenance": {
+    "enabled": false
+  },
+  "commitMessageSuffix": "J:KIT-282",
```

Comment:
why the suffix?

Reply:
The suffix allows the deployment pipeline to associate the merge commit with a jira, which prevents it from blocking the release.

Ref: https://coveord.atlassian.net/wiki/spaces/CM/pages/1786708612/Package+Rollout+Certifier#Changesets-validation-rules